app/react/Entities/components/EntityViewer.js

```diff
@@ -191,13 +192,23 @@ export class EntityViewer extends Component {
             >
               <ul className="nav nav-tabs">
                 <li>
-                  <TabLink to="info">
+                  <TabLink
+                    to="info"
+                    role="button"
+                    tabIndex="0"
+                    aria-label={t('System', 'Info', null, false)}
```

Comment:
already on screen on the span child.

Reply:
The span text does not work is because the `tooltip` class gives it a display: none style rule which is then omitted by the screen reader. Easier solution is to make the parent div focusable using `tabIndex="0"` and registering it as a button then giving it an aria label.